Patches/PlayerControlPatch.cs

```diff
@@ -2698,14 +2700,16 @@ public static void Postfix(PlayerControl __instance)
                             Mini.Age += 1;
                             Mini.GrowUpTime = 0;
                             Logger.Info($"年龄增加1", "Child");
-                            player.SetKillCooldown();
 
                             if (Mini.UpDateAge.GetBool())
                             {
-                                Mini.SendRPC();
-                                Utils.NotifyRoles();
+                                foreach (var pc in Main.AllPlayerControls)
+                                {
+                                    if (pc.PlayerId != player.PlayerId) continue;
+                                    player.Notify(GetString("MiniUp"));
+                                }
```

Comment:
I don't think it's necessary to run foreach

You can just write:
`if (player.Is(CustomRoles.Nice/EvilMini)) 
player.Notify(GetString("MiniUp"));`

Reply:
It's done. _.